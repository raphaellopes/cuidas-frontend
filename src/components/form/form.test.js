// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import Form from './index';

const component = props => (renderer.create(
  <Form {...props} />,
));

describe('<Form />', () => {
  test('Should render the component', () => {
    const tree = component({ children: 'My children' }).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
