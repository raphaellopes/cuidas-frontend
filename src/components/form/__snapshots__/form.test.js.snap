// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> Should render the component 1`] = `
<form>
  My children
</form>
`;
